docs/output-logstash.asciidoc

```diff
@@ -125,23 +122,38 @@ output plugins.
 
 &nbsp;
 
-[id="plugins-{type}s-{plugin}-host"]
-===== `host`
+[id="plugins-{type}s-{plugin}-hosts"]
+===== `hosts`
 
-* Value type is a <<string,string>> ip address or hostname
-* There is no default value
+* Value type is <<string,string>>
+* There is no default value for this setting.
+* Constraints:
+** When using IPv6, IP address must be in an enclosed in brackets.
+** When a port is not provided, the default `9800` is used.
 
-Specify which host to connect to by providing its ip address or resolvable hostname.
+A downstream input {ls} host or IP address to connect.
 
-NOTE: when using SSL, the server that responds must present a certificated with identity claim matching this host name or ip address.
+NOTE: Single host endpoint is supported for `hosts`. Multi-host support is coming soon.
 
-[id="plugins-{type}s-{plugin}-port"]
-===== `port`
+Host can be any of IPv4, IPv6 (in enclosed bracket) or host name, examples:
 
-* Value type is a <<number,number>> port
-* There is no default value
+      `"127.0.0.1"
+      `"127.0.0.1:9800"`
+      `"foo-bar.com"`
+      `"foo-bar.com:9800"`
+      `"[::1]"`
+      `"[::1]:9000"`
+      `"[2001:0db8:85a3:0000:0000:8a2e:0370:7334]"`
```

Comment:
1. An asciidoc list renders better than the block
2. use `*.example.com` instead of `foo-bar.com`
3. missing variant with bracketed uncompressed ipv6 with port

```suggestion
* `"127.0.0.1"`
* `"127.0.0.1:9801"`
* `"ds.example.com"`
* `"ds.example:9802"`
* `"[::1]"`
* `"[::1]:9803"`
* `"[2001:0db8:85a3:0000:0000:8a2e:0370:7334]"`
* `"[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:9804"`
```